Client/Frontend/Browser/SwipeAnimator.swift

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+struct SwipeAnimatorUX {
+    var totalRotationInDegrees: Double
+    var deleteThreshold: CGFloat
+    var totalScale: CGFloat
+    var totalAlpha: CGFloat
+    var minExitVelocity: CGFloat
+    var recenterAnimationDuration: NSTimeInterval
+}
+
+private let DefaultSwipeAnimatorUX =
+    SwipeAnimatorUX(
```

Comment:
Do we plan on having multiple `SwipeAnimatorUX`s? This abstraction seems like overkill...I'd just bake these constants directly into the `struct` like we do in other places, unless we have a good reason not to.